strings/strings.json

```diff
@@ -2552,16 +2552,16 @@
   "tech_mass_relay_effect": "The Mass Relay is a structure that temporarily lowers the mass of a starship allowing it to accelerate to higher speeds. This will make ships departing your shipyard travel thrice as fast, but they'll have no assistance on subsequent journeys unless returned to the shipyard.",
   "tech_nav_data": "Navigation Data",
   "tech_nav_data_effect": "Decode navigation data from the derelict ship to see where it's been.",
-  "tech_nav_data_result": "Navigation logs show the ship passed through the Tau Ceti system thousands of years ago, as well as it scanned several interesting sites in the Kuiper Belt and the dwarf planet %0.",
+  "tech_nav_data_result": "Navigation logs show the ship passed through the Tau Ceti system thousands of years ago. It also scanned several interesting sites in the Kuiper Belt and the dwarf planet %0.",
   "tech_sensor_logs": "Tau Ceti Data",
   "tech_sensor_logs_effect": "Analyze the Tau Ceti Sensor Data",
-  "tech_sensor_logs_result": "Sensor logs indicate the Tau Ceti system contains two good candidate planets for an extrasolar colony, you don't however currently possess technology to reach there in any reasonable time frame.",
+  "tech_sensor_logs_result": "Sensor logs indicate the Tau Ceti system contains two good candidate planets for an extrasolar colony. However, you don't currently possess technology to reach there in any reasonable time frame.",
   "tech_dronewar": "Drone Warfare",
-  "tech_dronewar_effect": "The cost to send soldiers to invade %0 is too high, instead prepare a robot army commanded the by Titan AI to secure the dwarf planet.",
+  "tech_dronewar_effect": "The cost to send soldiers to invade %0 is too high, instead prepare a robot army commanded by the Titan AI to secure the dwarf planet.",
```

Comment:
Note: "the Titan AI" should have 'Titan' replaced with the current race's name for Titan. I didn't change this since it would require a code change to provide a new placeholder value.